audio/pulseaudio/files/patch-src_modules_macosx_module_coreaudio_device.c-import-util-h.diff

```diff
@@ -0,0 +1,10 @@
+--- src/modules/macosx/module-coreaudio-device.c.orig	2020-09-10 19:42:18.000000000 +0200
```

Comment:
This is a trivial patch, but is there an [Upstream-Status](https://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines#Patch_Header_Recommendations:_Upstream-Status) that could be added here?

Reply:
There is no upstream GitLab issue for this yet (in fact, there are no upstream issues at all, which seem quite odd). I have forked the upstream project, and my patch is currently going through upstream CI. Once I opened a merge request, I can add an upstream status.

Reply:
Are you looking in the right repository? The issue tracker appears to be here: https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues
It should be okay to use `Upstream-Status: Pending` for now if needed.

Reply:
The purpose of the `Upstream-Status` is to go in the patchfile rather than the commit message. It should use also use a precise link:
```diff
Upstream-Status: Submitted [https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/merge_requests/365]
+++ src/modules/macosx/module-coreaudio-device.c	2020-09-10 19:46:35.000000000 +0200
--- src/modules/macosx/module-coreaudio-device.c.orig	2020-09-10 19:42:18.000000000 +0200
@@ -27,6 +27,7 @@
 #include <config.h>
…
```

Reply:
Upstream issue filed, merge request submitted and CI jobs passed, commit message updated accordingly.

Reply:
> The purpose of the `Upstream-Status` is to go in the patchfile rather than the commit message. It should use also use a precise link:

That's not what the MacPorts documentation states:
```
Submitted [where]
  - Submitted to upstream, waiting approval
  - Optionally include where it was submitted, such as the author, mailing
    list, etc.
```
No need to be overly precise. 😉

Reply:
> The purpose of the `Upstream-Status` is to go in the patchfile rather than the commit message.

That was indeed an oversight on my end. Will fix.

Reply:
Linking to the exact PR, mailing list thread, etc. saves others from having to search for exactly where they can check its acceptance status. Using an intentionally imprecise link seems to defeat the purpose of using a link at all.

I presume that the docs say that mentioning "where" is optional because some upstream development is not done so openly/transparently; "submitted" could mean "I sent an email with the patch attached to the address listed in the project's README, and it hasn't bounced".

Reply:
Given that the patch is both trivial and an original work of mine (yay me!) first created for MacPorts and only later submitted upstream, it does not really matter much, does it? In any case, let it not be said that I am afraid of long URLs. I have updated the patch header and the commit messsge.

I hope this concludes our discussion, and I'd be glad to see this merge request merged. 😎